modin/experimental/xgboost/test/test_dmatrix.py

```diff
@@ -19,9 +19,17 @@
 import xgboost as xgb
 
 import modin.pandas as pd
+from modin.config import Engine
 import modin.experimental.xgboost as mxgb
 
 
+if Engine.get() == "Python":
```

Comment:
Our xgb implementation doesn't work on dask or mpi too for now. Why do we check for python only?

Reply:
This file is tested with Dask in the CI and the tests pass.

Reply:
As I see we tested this file only  with Ray previously but you enabled testing with Dask. Testing for this file passes with Dask because there are no tests specific to our xgb implementation but this may change. I would recommend to leave testing only with Ray.

Reply:
I can do it right now, but why not make changes the moment the tests fail?

Reply:
Because we explicitly say that we support xgb on ray only, but at the same time we test it on other engine, which accidendtly passes. Let's test only on Ray for now.

Reply:
The reason of the test pass is that we don't check for any xgb-related things in it. Anyway, I want to be explicit and test xgb only on the engines we explicitly support.

Reply:
Ok